### DISCUSSION TITLE
Space shooter?

### DISCUSSION THREAD

**Discussion by ajones1973** [jacksparrow124/beginner-python-games#2] [General]
hello! is the shooter coming out today? i know you mentioned it yesterday, but i just wanted to confirm.

**Comment by jacksparrow124** [comment]
Hello! today **_is_** the release of the space shooter. The .py file is the code, while the other two files are sound effects and graphics. please download these into the same folder as the game. 

also, where it says:
`img1 = PhotoImage(file =' #the path in which GRAPHIC effect 1 is in')`
`#same thing for the next line, just a diffrent effect`
`#same thing on the third line.`


but when it gets to sounds its a little diffrent:

 `def play_shoot():`
  `winsound.PlaySound('#your path of the first SOUND effect', 1)`  

then the same thing for the next two SOUND effects

please reach out in the discussions tab about issues with these lines or other parts of the code, as i am always happy to help.

sincerely,

                    jacksparrow124
